src/code/templates/fv-egg-game.js

```diff
@@ -880,7 +880,7 @@ export default class FVEggGame extends Component {
     // after the egg has hatched, reset gametes if the user selects another gamete
     if (hatchSoundPlayed) {
       resetAnimationEvents({ showStaticGametes: true, showHatchAnimation: false });
-      this.props.onResetGametes(true);
+      this.props.onResetGametes(false);
```

Comment:
I could have left in the change to reset `hatchSoundPlayed` but I decided to leave it alone